setup.py

```diff
@@ -14,7 +14,7 @@
         'pytest',
         'pytest-cov',
     ],
-    ':python_version < "3.4"': [
+    'python_version < "3.4"': [
```

Comment:
Turns out this was an environment issue and `setuptools` was not, in fact, being updated.